site/content/resources/compatibility-matrix.md

```diff
@@ -8,29 +8,29 @@ These combinations of versions are specifically tested in CI and supported by th
 
 ## Compatibility Matrix
 
-| Contour Version | Envoy Version        | Kubernetes Versions | Operator Version |
-| --------------- | :------------------- | ------------------- | ---------------- |
-| main            | [1.19.1][13]         | 1.22, 1.21, 1.20    | [main][50]       |
-| 1.18.1          | [1.19.1][13]         | 1.21, 1.20, 1.19    | [1.18.1][62]     |
-| 1.18.0          | [1.19.0][10]         | 1.21, 1.20, 1.19    | [1.18.0][61]     |
-| 1.17.2          | [1.18.4][12]         | 1.21, 1.20, 1.19    | N/A              |
-| 1.17.1          | [1.18.3][9]          | 1.21, 1.20, 1.19    | N/A              |
-| 1.17.0          | [1.18.3][9]          | 1.21, 1.20, 1.19    | [1.17.0][60]     |
-| 1.16.1          | [1.18.4][12]         | 1.21, 1.20, 1.19    | N/A              |
-| 1.16.0          | [1.18.3][9]          | 1.21, 1.20, 1.19    | [1.16.0][59]     |
-| 1.15.2          | [1.18.4][12]         | 1.21, 1.20, 1.19    | N/A              |
-| 1.15.1          | [1.18.3][9]          | 1.21, 1.20, 1.19    | [1.15.1][58]     |
-| 1.15.0          | [1.18.2][8]          | 1.21, 1.20, 1.19    | [1.15.0][57]     |
-| 1.14.2          | [1.17.4][11]         | 1.20, 1.19, 1.18    | N/A              |
-| 1.14.1          | [1.17.2][7]          | 1.20, 1.19, 1.18    | [1.14.1][56]     |
-| 1.14.0          | [1.17.1][6]          | 1.20, 1.19, 1.18    | [1.14.0][55]     |
-| 1.13.1          | [1.17.1][6]          | 1.20, 1.19, 1.18    | [1.13.1][54]     |
-| 1.13.0          | [1.17.0][5]          | 1.20, 1.19, 1.18    | [1.13.0][53]     |
-| 1.12.0          | [1.17.0][5]          | 1.19, 1.18, 1.17    | [1.12.0][52]     |
-| 1.11.0          | [1.16.2][4]          | 1.19, 1.18, 1.17    | [1.11.0][51]     |
-| 1.10.1          | [1.16.2][4]          | 1.19, 1.18, 1.17    | N/A              |
-| 1.10.0          | [1.16.0][3]          | 1.19, 1.18, 1.17    | N/A              |
-| 1.9.0           | [1.15.1][2]          | 1.18, 1.17, 1.16    | N/A              |
+| Contour Version | Envoy Version        | Kubernetes Versions | Operator Version | Gateway API Version |
+| --------------- | :------------------- | ------------------- | ---------------- | --------------------|
+| main            | [1.19.1][17]         | 1.22, 1.21, 1.20    | [main][50]       | v1alpha1            |
```

Comment:
Looks like you may have run into some merge conflict issues with the Envoy version link numbers here -- the ones you have are no longer valid, need to switch to what's in `main`.

Reply:
dang, thanks, I'll update.